pkg/registry/admissionregistration/validatingadmissionpolicy/storage/storage_test.go

```diff
@@ -225,3 +226,11 @@ func TestCategories(t *testing.T) {
 	expected := []string{"api-extensions"}
 	registrytest.AssertCategories(t, storage, expected)
 }
+
+var replicaLimitsResolver resolver.ResourceResolverFunc = func(gvk schema.GroupVersionKind) (schema.GroupVersionResource, error) {
```

Comment:
nit: `func replicaLimitsResolver(gvk schema.GroupVersionKind) (schema.GroupVersionResource, error)`

Reply:
But I can do `resolver.ResourceResolverFunc(replicaLimitsResolver)` at the call site, so I'll do that.